mpmp/src/main/Client.java

```diff
@@ -91,4 +91,11 @@ public static void broadcast(String s) {
 	public static void init() {
 		clients = new HashSet<>();
 	}
+	
+	public static Client search(String name) {
+		for (Client c : clients)
+			if (name.equals(c.getName()))
```

Comment:
Micro suggestion: can't you say `c.name` instead of` c.getName()`? We are in the same class.